host-bin/mount-chroot

```diff
@@ -60,22 +62,32 @@ addtrap "stty echo 2>/dev/null"
 
 # Function to prompt the user for a passphrase. Sets $passphrase.
 promptNewPassphrase() {
+    NOPASS=''
     echo -n "Choose an encryption passphrase for $NAME: " 1>&2
     [ -t 0 ] && stty -echo
-    while [ -z "$passphrase" ]; do
+    while [ -z "$passphrase" -a -z "$NOPASS" ]; do
         read -r passphrase
-        if [ -z "$passphrase" ]; then
```

Comment:
The point of the `-e` flag is to make your chroot install more secure.  If we allow blank passwords without further configuration, then people will assume that even with a blank passphrase, things are fine (they will ignore the later warnings).  Unless you see actual value in allowing co-located unencrypted passphrases, I'd rather disallow it to avoid any chance of a false sense of security.